code/part-two/processor/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "homepage": "https://github.com/hyperledger/education-cryptomoji#readme",
   "dependencies": {
+    "npm": "^5.8.0",
```

Comment:
interesting, I didn't know you could/had to specify a specific `npm` version as a dependency! would the currently globally installed `npm` then install a local v5.8+ of `npm` in the `node_modules/`, which is then subsequently used for all npm & npm script commands? 🤔